web-registry/src/utils/price/computeMedianPrice.ts

```diff
@@ -1,15 +1,32 @@
-type Params = {
-  prices: number[];
-};
+export interface Order {
+  quantity: number;
+  usdPrice: number;
+}
+
+export const calculateMedianPrice = (orders: Order[]): number => {
```

Comment:
we could probably keep the name `computeMedianPrice` to be more consistent with the file name